src/api/Animated/Easing.js

```diff
@@ -89,14 +87,7 @@ class Easing {
     return 7.5625 * t * t + 0.984375;
   }
 
-  static bezier(x1, y1, x2, y2, epsilon) {
-    if (epsilon === undefined) {
-      // epsilon determines the precision of the solved values
-      // a good approximation is:
-      var duration = 500; // duration of animation in milliseconds.
-      epsilon = (1000 / 60 / duration) / 4;
-    }
-
+  static bezier(x1, y1, x2, y2, epsilon = 0.008333) {
```

Comment:
might be better to use `epsilon = 1000 / 60 / 500 / 4`, since someone could be depending on a more accurate decimal value somewhere (unlikely, but still).